src/ortc.cpp

```diff
@@ -103,7 +103,8 @@ namespace mediasoupclient
 				MSC_THROW_TYPE_ERROR("missing codec.mimeType");
 
 			std::smatch mimeTypeMatch;
-			std::regex_match(mimeTypeIt->get<std::string>(), mimeTypeMatch, MimeTypeRegex);
+			std::string tmp = mimeTypeIt->get<std::string>();
+			std::regex_match(tmp, mimeTypeMatch, MimeTypeRegex);
```

Comment:
Please avoid cosmetic changes in a PR that is not about that.

Reply:
There was a change going from C++11 to C++14 where std::regex_search is no longer allowed to take a r-value

Reply:
Thanks for this, although libmediasoupclient uses C++11:

`set(CMAKE_CXX_STANDARD 11)`

Reply:
I was unable to build libmediasoup with m81 without switching to C++14

Reply:
> Thanks for this, although libmediasoupclient uses C++11:
> 
> `set(CMAKE_CXX_STANDARD 11)`

Hi~ The latest version of google libwebrtc uses the C++ 14 Digit Separator syntax to make large numbers easier for human readers.
So if you use C++ 11, cmake will always fail.

```
/xxx/google/webrtc/src/api/units/time_delta.h:43:26: warning: 
      missing terminating ' character [-Winvalid-pp-token]
    return FromFraction(1'000, value);
                         ^
/xxx/google/webrtc/src/api/units/time_delta.h:86:9: error: 
      use of undeclared identifier 'RelativeUnit'
  using RelativeUnit::RelativeUnit;
        ^
/xxx/google/webrtc/src/api/units/time_delta.h:38:26: error: 
      expected ')'
    return FromFraction(1'000'000, value);
                         ^
/xxx/google/webrtc/src/api/units/time_delta.h:38:24: note: 
      to match this '('
    return FromFraction(1'000'000, value);
                       ^
/xxx/google/webrtc/src/api/units/time_delta.h:36:30: error: 
      no return statement in constexpr function
  static constexpr TimeDelta Seconds(T value) {
                             ^
/xxx/google/webrtc/src/api/units/time_delta.h:43:26: error: 
      expected ')'
    return FromFraction(1'000, value);

```

The versatica/mediasoup-broadcaster-demo project also has this problem.

Reply:
We'll check it during the coming days.